.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -6,7 +6,6 @@ channels:
 dependencies:
   - pytest
   - pytest-cov
```

Comment:
@NicolasHug @pmeier Would you happen to know if the `pytest-cov` can also be removed?

Reply:
It's different.